perllib/FixMyStreet/App/Controller/Waste/Echo.pm

```diff
@@ -108,6 +108,15 @@ sub receive_echo_event_notification : Path('/waste/echo') : Args(0) {
                 $updates->suppress_alerts(1);
             }
 
+            # If it's a cancellation update on a container request
+            # that is already cancelled, don't send an alert as
+            # we would have already sent one out for this.
+            if ($p->category eq 'Request new container'
+                && $request->{status} eq 'cancelled'
+                && $p->state eq 'cancelled') {
+                $updates->suppress_alerts(1);
```

Comment:
Just checking (I think we discussed this, but have forgotten) - this is just in case it's not already caught/hidden by the 'identical update' checking (now that we use the template)? Do we want to only suppress the alert, or if we're here, skip the update entirely?